docs/src/content/docs/es/reference/route-data.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Referencia de los Datos de Ruta
+description: La documentación de referencia completa para el objeto de datos de ruta de Starlight.
+---
+
+El objeto de datos de ruta de Starlight contiene información sobre la página actual.
+Aprende más sobre cómo funciona el modelo de datos de Starlight en la guía de [“Datos de Ruta”](/es/guides/route-data/).
+
+En los componentes de Astro, accede a los datos de ruta desde `Astro.locals.starlightRoute`:
+
+```astro {4}
+---
+// src/components/Custom.astro
+
+const { hasSidebar } = Astro.locals.starlightRoute;
+---
+```
+
+En el [middleware de ruta](/guides/route-data/#customizing-route-data), accede a los datos de ruta desde el objeto de contexto pasado a tu función de middleware:
```

Comment:
We have a rule to always link to `/es/` links inside `/es/` pages to avoid moving a reader out of their chosen language:

```suggestion
En el [middleware de ruta](/es/guides/route-data/#customizing-route-data), accede a los datos de ruta desde el objeto de contexto pasado a tu función de middleware:
```